webpack.config.js

```diff
@@ -1,10 +1,19 @@
-var webpack = require("webpack");
+const webpack = require('webpack')
```

Comment:
@jkdf2 Can we make single quotes our convention and set a rule in the ESlint config?

Reply:
@mbowen13 All style is handled through Prettier and yes it is possible to set our project to single quotes, but this is inadvisable especially in combination with JSX. Please see https://github.com/prettier/prettier/blob/master/Rationale.md